bastionzero/service/autodiscoveryscripts/bzero.go

```diff
@@ -48,6 +48,31 @@ func (s *AutodiscoveryScriptsService) GetBzeroBashAutodiscoveryScript(ctx contex
 	return script, resp, nil
 }
 
+// GetBzeroPowershellAutodiscoveryScript gets a powershell script that can be used to
+// install the latest production BastionZero agent (bzero) on your Windows targets.
+//
+// BastionZero API docs: https://cloud.bastionzero.com/api/#get-/api/v2/autodiscovery-scripts/bzero/bash
+func (s *AutodiscoveryScriptsService) GetBzeroPowershellAutodiscoveryScript(ctx context.Context, opts *BzeroBashAutodiscoveryOptions) (*BzeroBashAutodiscoveryScript, *http.Response, error) {
```

Comment:
Can you make a separate `opts` type and response object for this endpoint/func instead of reusing `BzeroBashAutodiscoveryOptions` and `BzeroBashAutodiscoveryScript`? The reason being in case the `/windows/powershell` endpoint ever adds additional `GET` query options in the future that aren't applicable to the bash one, or if the response object ever diverges from the one used for the bash AD script endpoint.

You can just copy these types and change the names and comments accordingly (`BzeroPowershellAutodiscoveryOptions` and `BzeroPowershellAutodiscoveryScript`):

https://github.com/bastionzero/bastionzero-sdk-go/blob/ec69b6a59b7fba141842f35713caa06371468502/bastionzero/service/autodiscoveryscripts/bzero.go#L11-L24

Then just change `script := new(BzeroBashAutodiscoveryScript)` to `script := new(BzeroPowershellAutodiscoveryScript)` and the `opts` arg to the new type as well.